charts/karpenter/templates/clusterrole.yaml

```diff
@@ -36,3 +36,13 @@ rules:
   - apiGroups: ["karpenter.k8s.aws"]
     resources: ["ec2nodeclasses", "ec2nodeclasses/status"]
     verbs: ["patch", "update"]
+{{- if .Values.webhook.enabled }}
```

Comment:
We don't need these permissions, since the validation and defaulting webhooks are removed

Reply:
Ops didn't see @njtran made the comments